apps/test/unit/code-studio/components/SoundListEntryTest.js

```diff
@@ -59,4 +61,17 @@ describe('SoundListEntry', () => {
     // First child is a icon control for pause and play
     assert.equal(wrapper.props().children[0].props.children.props.className, 'fa fa-pause-circle fa-2x');
   });
+
+  it('stops playing the sound when deselected', () => {
+    const wrapper = shallow(
+      <SoundListEntry
+        {...defaultProps}
+      />
+    );
+    sounds.stopPlayingURL = sinon.spy();
```

Comment:
This probably isn't right.  Since `stopPlayingURL` already exists, you should wrap it instead of overwriting it.

```js
// If you still want the original function to get called, you should wrap it in a spy:
sinon.spy(sounds, 'stopPlayingURL');

// If you don't want the original function to get called, you should use a stub:
sinon.stub(sounds, 'stopPlayingURL');

// Either way, you should also clean this up at the end of your test:
sounds.stopPlayingURL.restore();
```